Makefile

```
@@ -92,7 +92,7 @@ generate-renovate-ignore-deps:
 
 .PHONY: docker-images
 docker-images:
-	@docker build -t $(CERT_IMAGE_REPOSITORY):$(IMAGE_TAG) -t $(CERT_IMAGE_REPOSITORY):latest -f Dockerfile --target cert-controller-manager .
+	@docker build --platform linux/amd64,linux/arm64 -t $(CERT_IMAGE_REPOSITORY):$(IMAGE_TAG) -t $(CERT_IMAGE_REPOSITORY):latest -f Dockerfile --target cert-controller-manager .
```

I don't think we need a `--platform` argument here. Anyway it does not seem to work as expected. Probably a `docker buildx build` would be needed to fix it. But we don't want cross compilation as it is quite slow. The change in the pipeline_definitions should be enough, I guess.

```suggestion
	@docker build -t $(CERT_IMAGE_REPOSITORY):$(IMAGE_TAG) -t $(CERT_IMAGE_REPOSITORY):latest -f Dockerfile --target cert-controller-manager .
```

---

Reverted the change and kept only the small adaption in the `pipeline_definitions` :)